client/src/components/AddNewProfile.js

```diff
@@ -16,46 +17,96 @@ const AddNewProfile = () => {
     support_type: "",
     profile_type: "",
     status: "new",
-    join_date: new Date()
+    nationality: "",
+    join_date: new Date(),
   });
-
+  const [errors, setErrors] = useState([]);
   const [profileCreated, setProfileCreated] = useState(null);
+  const [nationality, setNationality] = useState([]);
```

Comment:
```suggestion
  const [nationalities, setNationalities] = useState([]);
```
Since we are storing a list of nationalities, let's make the variable name plural

Reply:
This is my fault. I accidentally thought this was the nationality state for the profile in an earlier review. My brain obviously wasn't functioning properly at the time. Sorry, @NataliaCoder!